test/e2e/frontend/cypress/tests/instances/version-history/snapshots.spec.js

```diff
@@ -48,11 +49,14 @@ describe('FlowForge - Instance Snapshots', () => {
         cy.visit(`/instance/${projectId}/version-history/snapshots`)
         cy.wait('@snapshotData')
 
+        // eslint-disable-next-line cypress/require-data-selectors
         cy.get('button[data-action="create-snapshot"]').click()
 
         cy.get('[data-el="dialog-create-snapshot"]').should('be.visible')
+        // eslint-disable-next-line cypress/require-data-selectors
         cy.get('.ff-dialog-header').contains('Create Snapshot')
```

Comment:
`data-el="create-client-dialog"` exists on the dialog, this should be included here as a selector